chainerx_cc/chainerx/routines/loss.h

```diff
@@ -15,6 +15,17 @@ Array HuberLoss(const Array& x1, const Array& x2, Scalar delta);
 
 Array SigmoidCrossEntropy(const Array& x1, const Array& x2);
 
+enum class SoftmaxCrossEntropyReduceMode {
+    kMean = 1,
+    kNo,
+};
+
+Array SoftmaxCrossEntropy(
+        const Array& x1,
+        const Array& x2,
+        bool normalize = true,
+        SoftmaxCrossEntropyReduceMode reduce_mode = SoftmaxCrossEntropyReduceMode::kMean);
```

Comment:
Following [this interface design principle](https://github.com/chainer/chainer/wiki/Development-policies#api-1), we can probably skip the `normalize` and `reduce_mode` altogether?

Reply:
Sure, I removed them!